modules/nf-core/ashlar/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: "ashlar"
+description: Alignment by Simultaneous Harmonization of Layer/Adjacency Registration
+keywords:
+  - image_processing, alignment, registration
+tools:
+  #
+  - "ashlar":
+      description: "Alignment by Simultaneous Harmonization of Layer/Adjacency Registration"
+      homepage: "https://labsyspharm.github.io/ashlar/"
+      documentation: "https://labsyspharm.github.io/ashlar/"
+      doi: "10.1101/2021.04.20.440625"
+      licence: "['MIT']"
+
+input:
+  #
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', : args: '--flip-x' ]
+  - file_in:
+      type: path
+      description: multidimentional image data in formats from most commercial microscopes
+      pattern: "*.ome.tiff"
+
+output:
+  #
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', : args: '--flip-x' ]
+  - file_out:
+      type: file
+      description: pyramidal OME-TIFF
+      pattern: "*.tif"
```

Comment:
This should be `*.ome.tif`.

Reply:
I set it as `*.tif` because the output can be `.tif` if there's a non-ome.tif input.  That said I realize that the input should be changed to `*.tif` because of this.  I'll make that change.
Thank you for pointing this out.

Reply:
tests passed as expected with this change, so I've pushed the commit.

Reply:
As I commented above, the input name does not affect the output name. The output are only non-.ome.tif if the user puts `{cycle}` and/or `{channel}` in the _output_ file path (-o). I am tempted to just not support this option in the nf-core module. If someone wants to split their all-in-one ome-tiff into single channels or whatever, they can use a separate module to post-process it. mcmicro will always use the all-in-one ome-tiff so we don't need the single-channel tif output anyway. I don't think we need to expose the `-o` option to module users at all, actually.

Reply:
Therefore the file_out pattern should be `*.ome.tif`.

Reply:
Sounds good. updated, tested and pushed